cmd/local/local.go

```diff
@@ -61,11 +61,7 @@ func NewCommand() *cobra.Command {
 	// todo: get it from GH token , use it for console
 	localCmd.Flags().StringVar(&adminEmail, "admin-email", "", "the email address for the administrator as well as for lets-encrypt certificate emails")
 	localCmd.Flags().StringVar(&metaphorBranch, "metaphor-branch", "main", "metaphor application branch")
-	// todo: UPDATE IT BEFORE MERGING
-	// todo: UPDATE IT BEFORE MERGING
-	// todo: UPDATE IT BEFORE MERGING
-	// todo: UPDATE IT BEFORE MERGING
-	localCmd.Flags().StringVar(&gitOpsBranch, "gitops-branch", "", "version/branch used on git clone")
+	localCmd.Flags().StringVar(&gitOpsBranch, "gitops-branch", "main", "version/branch used on git clone")
```

Comment:
this need to be empty afte the changes of branch.. to default to tag. please use branch flag for local testing.

Reply:
It is not a good practice to hard-code this branch as we use the tag as the main driver for final releases.